test_scripts/RC/OnRCStatus/023_notification_by_rc_functionality_allowed_on_hmi_with_registered_app.lua

```diff
@@ -0,0 +1,52 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- [SDL_RC] TBD
+--
+-- Description:
+-- In case:
+-- 1) RC functionality is disallowed on HMI
+-- 2) RC application is registered
+-- 3) RC functionality is allowed on HMI
+-- SDL must:
+-- 1) Send OnRCStatus notification with allowed = true to registered mobile application and the HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/OnRCStatus/commonOnRCStatus')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function disableRCFromHMI()
+	common.getHMIconnection():SendNotification("RC.OnRemoteControlSettings", { allowed = false })
+	common.wait(2000)
+end
+
+local function enableRCFromHMI()
+	local pModuleStatus = {
+    freeModules = common.getModulesArray(common.getAllModules()),
+    allocatedModules = { },
+    alowed = true
+  }
+	common.getHMIconnection():SendNotification("RC.OnRemoteControlSettings", { allowed = true })
+	common.validateOnRCStatusForApp(1, pModuleStatus)
+	EXPECT_HMINOTIFICATION("RC.OnRCStatus")
+	:Times(0)
```

Comment:
@GetmanetsIrina in description : SDL must send OnRCStatus notification with allowed = true to registered mobile application and the HMI

Do we need Times(0) here?